io/EptReader.cpp

```diff
@@ -704,6 +691,17 @@ uint64_t EptReader::readZstandard(PointView& dst, const Key& key,
 
     ShallowPointTable table(*m_remoteLayout,uncompressed.data(),
             uncompressed.size());
+    return processBinaryTable(dst, nodeId, table);
+#else
+    throwError("Cannot read Zstandard dataType: "
+        "PDAL must be configured with WITH_ZSTD=On");
+    return 0;   // Suppress "control reaches end of non-void function" warning.
+#endif
+}
+
+u_int64_t EptReader::processBinaryTable(PointView& dst, const uint64_t nodeId,
```

Comment:
Should be `uint64_t` instead of `u_int64_t` I'd think.  Same for the header version.

Reply:
Good point, fixed now